packages/migrate/migrations/routes/migrate_page_server/index.spec.js

```diff
@@ -5,7 +5,7 @@ import { migrate_page_server } from './index.js';
 
 for (const sample of read_samples(import.meta.url)) {
 	test(sample.description, () => {
-		const actual = migrate_page_server(sample.before, sample.filename);
+		const actual = migrate_page_server(sample.before, sample.filename ?? '+page.server.js');
```

Comment:
Should this defaulting be done inside the method? I.e. make the second parameter optional

Reply:
this code is saying that all samples implicitly _do_ have a filename, `+page.server.js`, unless one is specified. making the parameter optional for the sake of the tests is the wrong solution